tests/metricsSubmissionUrlTests.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { getEndpointParams } from '../source/utils';
+
+describe('Test Submission URL information in metrics support', () => {
+  describe('Test universe name', () => {
```

Comment:
We can remove this description since we don't use it and only makes test less clear.